README.md

```diff
@@ -536,6 +536,7 @@ Table of Contents
    * [repl.it](https://repl.it/) — A cloud coding environment for various program languages
    * [codesandbox.io](https://codesandbox.io/) — Online React playground
    * [stackblitz.com](https://stackblitz.com/) — Online VS Code IDE for Angular & React
+   * [cacher.io](https://www.cacher.io) - Collaborative code snippet organizer with labels, sync for GitHub Gist and extensions for VSCode, Atom and Sublime.
```

Comment:
some of these features are not in the free edition, please update with the correct free features